src/Globe/TileMesh.js

```diff
@@ -46,7 +48,9 @@ function TileMesh(params, builder, geometryCache) {
     this.geometry = defaultValue(geometryCache, new TileGeometry(params, builder));
     this.normal = params.center.clone().normalize();
 
-    this.distance = params.center.length();
+    var worldNormal = new THREE.Vector3(0, 0, 1).applyQuaternion(this.OBB().getWorldQuaternion());
```

Comment:
Ok
It's possible to move compute normal in `TileGeometry` and implicitly in `BuilderEllipsoidTile` and `BuilderPlanarTile`. I don't understand why you need to project center to compute distance.